src/grt/grt-avhpi.adb

```diff
@@ -27,6 +27,19 @@ with Grt.Vstrings; use Grt.Vstrings;
 with Grt.Rtis_Utils; use Grt.Rtis_Utils;
 with Grt.To_Strings;
 
+-- Acronyms that I don't understand:
+   -- Rti
+   -- Rtis
+   -- Rtin
+
+
+-- Questoins:
+   -- Explanation for the Ctxt objects would be really helpful.
+   -- Why are they used, and what do they represent?
```

Comment:
Example for a port:
you have an rti that describes the port: its name, its type, its mode...
But you also need its address to get its value.  But a port is part of an entity that can be instantiated many times.  So you need the context of the port, which consists of the entity/architecture and the address of the entity/architecture instance.